specification/compute/resource-manager/Microsoft.Compute/stable/2020-05-01/disk.json

```diff
@@ -1889,6 +2158,149 @@
       ],
       "description": "The List disk encryption set operation response."
     },
+    "PrivateLinkServiceConnectionState":{
+      "properties":{
+        "actionsRequired": {
+          "type": "string",
+          "readonly": true,
+          "description": "Action required by the network admin."
+        },
+        "description": {
+          "type": "string",
+          "description": "description about the private link."
+        },
+        "status": {
+          "type": "string",
+          "description": "current state of the private link.",
+          "enum": [
+            "Approved",
+            "Rejected",
+            "Pending"
+          ],
+          "x-ms-enum": {
+            "name": "PrivateLinkServiceConnectionStates",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "PrivateEndpoint":{
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description":"Resource id of the private endpoint."
+        }
+      }
+    },
+    "PrivateEndpointConnectionProperties": {
+      "properties": {
+        "provisioningState":{
+          "readOnly": true,
+          "type": "string",
+          "description": "The private endpoint connection provisioning state."
+        },
+        "privateEndpoint":{
+          "readOnly": true,
+          "$ref": "#/definitions/PrivateEndpoint",
+          "description": "The private endpoint which owns the private endpoint connection."
+        },
+        "privateLinkServiceConnectionState":{
+          "$ref": "#/definitions/PrivateLinkServiceConnectionState"
+        }
+      }
+    },
+    "PrivateEndpointConnection": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/PrivateEndpointConnectionProperties"
+        },
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource Id of the private endpoint connection."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource name of the private endpoint connection."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type of the private endpoint connection."
+        }
+      }
+    },
+    "DiskAccessProperties": {
+      "properties": {
+        "privateEndpointConnections": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/PrivateEndpointConnection"
+          },
+          "description": "A read only collection of private endpoint connections on the diskAccess."
+        },
+        "provisioningState": {
```

Comment:
make it enum and add the possible values.

Reply:
will do